autospec/config.py

```diff
@@ -307,7 +307,10 @@
     (r"Add the installation prefix of \"(.*)\" to CMAKE_PREFIX_PATH", 0, None),
     (r"^.*\"(.*)\" with any of the following names.*$", 0, None),
     (r"module not found: .*#(.*);(?:.*)", 0, 'maven'),
-    (r"-- artifact (?:.*)#(?:.*);(?:.*)!(.*).jar", 0, 'maven')]
+    (r"-- artifact (?:.*)#(?:.*);(?:.*)!(.*).jar", 0, 'maven'),
+    (r"\[WARNING\] Unable to get resource '(?:.*):(.*):[jar|pom]*:(?:.*)' from repository", 0, 'maven'),
+    (r"\> Could not find (?:.*):(.*):(?:.*)\.$", 0, 'maven'),
+    (r"\> Could not resolve (?:.*):(.*):(?:.*)\.$", 0, 'maven')]
```

Comment:
We can get org and version out of these here, too, to generate a full `mvn({org}:{name}:jar) = {ver}`. I can submit a separate PR to get that.

Reply:
sounds good to me, please do :)

Reply:
Does gradle print the group id, artifact id, version, and (optional) classifier in the same way as maven? If not, then the new capture group handling in `failed_pattern()` will need further refactoring to support the gradle messages.

Reply:
`group:artifact:version`

I haven't seen classifier (or e.g. `pom`, `jar`, etc.)